client/src/Google_API/geolocation.jsx

```diff
@@ -4,7 +4,7 @@ import axios from 'axios';
 import ReactDOM from "react-dom";
 
 const geoConverter = function(zipcode) {
-  let token = '';
+  let token = 'AIzaSyCZBlCg07Jx_8Sp3ee1EDbC6wH2rgdzJcw';
```

Comment:
Looks like the token that I pushed accidently got into your code as well 😬 

It has already been escalated, but this token should probably be removed